eng/versioning/external_dependencies.txt

```diff
@@ -146,6 +146,7 @@ org.revapi:revapi-java;0.20.0
 org.revapi:revapi-maven-plugin;0.11.2
 
 
+
```

Comment:
nit: if the only changes in this file are empty newline additions then please push another PR that excludes this file